pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools", "wheel", "torch", "numpy==1.26.4"]  # PEP 508 specifications.
```

Comment:
I guess `pip install .` would trigger this, is that true? Would this get triggered if I run `python setup.py`? 

Also `numpy==1.26.4` seems too constrained than necessary. We probably don't need to specify numpy version for the gsplat library?